Pages/HomePage.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Marsqa1Specflow.Utilities;
+using OpenQA.Selenium;
+
+namespace Marsqa1Specflow.Pages
+{
+    public class HomePage
+    {
+        private readonly IWebDriver _driver;
+
+        public HomePage(IWebDriver driver)
+        {
+            this._driver = driver;
+        }
+
+        // By Locators
+        public static By BtnSignInBy = By.XPath("//a[@class='item'][(text()='Sign In')]");
+        public static By FieldEmailBy = By.XPath("//input[@name='email']");
+        public static By FieldPasswordBy = By.XPath("//input[@name='password']");
+        public static By BtnLoginBy = By.XPath("//button[@class='fluid ui teal button']");
+
+        // Text
+        public readonly string emailValid = "one@test.com";
+        public readonly string passwordValid = "Password1.";
+
+        // Methods
+        public void ClickSignIn()
+        {
+            WaitUtil.WaitVisible(_driver, BtnSignInBy).Click();
+        }
+
+        public void Login()
+        {
+            ClickSignIn();
+            _driver.FindElement(FieldEmailBy).SendKeys(emailValid);
+            _driver.FindElement(FieldPasswordBy).SendKeys(passwordValid);
```

Comment:
_driver.FindElement you could do this 
public static By FieldEmailBy = _driver.FindElement(By.XPath("//input[@name='email']"));

this way you then just do FieldEmailBy.SendKeys(emailValid); please change this everywhere

Reply:
Refactor code accross project to use IWebElement instead of fields of type By, e.g.:
public IWebElement FieldEmail => driver.FindElement(By.XPath("//input[@name='email']"));